src/clojure_ttt/board.clj

```diff
@@ -42,4 +42,11 @@
 
 (defn valid-position? [board position]
   (contains? (vec (available-positions board)) position))
-    
+
+(defn find-turn [board]
```

Comment:
you add find-turn and last-move in this commit, but where is the corresponding spec?